timesketch/models/sketch.py

```diff
@@ -431,19 +436,62 @@ def __init__(self, name, description, agg_type, parameters, chart_type,
             chart_type (str): Chart plugin type
             user (User): The user who created the aggregation
             sketch (Sketch): The sketch that the aggregation is bound to
-            view (View): Optional: The view that the aggregation is bound to
+            view (View): Optional, the view that the aggregation is bound to
+            aggregationgroup (AggregationGroup): Optional, an AggregationGroup
+                that the aggregation is bound to.
         """
         super(Aggregation, self).__init__()
         self.name = name
         self.description = description
         self.agg_type = agg_type
+        self.aggregationgroup = aggregationgroup
         self.parameters = parameters
         self.chart_type = chart_type
         self.user = user
         self.sketch = sketch
         self.view = view
 
 
+class AggregationGroup(
+        AccessControlMixin, LabelMixin, StatusMixin, CommentMixin, BaseModel):
+    """Implements the Aggregation Group model."""
+    name = Column(Unicode(255))
+    description = Column(UnicodeText())
+    aggregations = relationship(
+        'Aggregation', backref='aggregationgroup', lazy='select')
+    parameters = Column(UnicodeText())
+    how = Column(Unicode(15))
```

Comment:
You can add some more chars here. Not much impact on the DB, but will future proof it a bit if we need to add more "hows" later.

Reply:
ah... well..I can add a bit more, (i did change this to orientation as well).... if altair/vega decides to add a new orientation that is longer